sigs.yaml

```diff
@@ -978,13 +986,6 @@ sigs:
   - name: cluster-api-provider-azure
     owners:
     - https://raw.githubusercontent.com/kubernetes-sigs/cluster-api-provider-azure/master/OWNERS
-    meetings:
-    - description: Cluster API Azure meeting
-      day: Thursday
-      time: "13:30"
-      tz: PT (Pacific Time)
-      frequency: bi-weekly
-      url: https://docs.google.com/document/d/1P2FrRjuCZjGy0Yh72lwWCwmXekSEkqliUVTmJy_ETIk/edit?usp=sharing
```

Comment:
maybe we should start enumerating meetings under the subprojects like it was ^ here and currently for:
https://github.com/kubernetes/community/blob/d981c5d574201225f0c96537b8d0e59049aacd42/sig-cluster-lifecycle/README.md#cluster-api-provider-vsphere

and only have the main SIG meeting in the main list here:
https://github.com/kubernetes/community/blob/d981c5d574201225f0c96537b8d0e59049aacd42/sig-cluster-lifecycle/README.md#meetings

@castrojo @nikhita @justinsb @fabriziopandini WDYT?

Reply:
but i can do that in a follow up refactor.

/lgtm
/assign @castrojo 
PTAL for approval.